src/test/java/com/epam/test_generator/dao/SuitDAOTest.java

```diff
@@ -196,9 +192,34 @@ public void testUpdateName() {
         Assert.assertEquals(newSuit, suitDAO.getOne(id));
     }
 
+    @Test
+    public void testUpdateRowNumber() {
+
+        suitDAO.save(Arrays.asList(
+                new Suit(null, "1", "1", new ArrayList<>(), 1, new HashSet<>(), 1),
+                new Suit(null, "2", "2", new ArrayList<>(), 2, new HashSet<>(), 2),
+                new Suit(null, "3", "3", new ArrayList<>(), 3, new HashSet<>(), 3)
+        ));
+
+        final Map<Long, Integer> patch = new HashMap<Long, Integer>() {{
+            put(1L, 2);
+            put(2L, 6);
+            put(3L, 5);
+        }};
+
+        final List<Suit> suits = suitDAO.findByIdInOrderById(patch.keySet());
+
+        for (Suit suit : suits) {
+            suit.setRowNumber(patch.get(suit.getId()));
+        }
+
+        final List<Suit> suitsAfterChange = suitDAO.save(suits);
+        Assert.assertEquals(suits, suitsAfterChange);
```

Comment:
It is better to assert row numbers here instead of entire objects. It would highlight real job of test case.

Reply:
Done